src/wazuh_modules/syscollector/syscollector_windows.c

```diff
@@ -1028,6 +1053,63 @@ void list_hotfixes(HKEY hKey, int usec, const char *timestamp, int ID, const cha
     }
 }
 
+// Retrieve the respective KB for those hotfixes which come as Rollup.
+// ex -> Package_for_RollupFix~31bf3856ad364e35~amd64~~18362.959.1.9
+char * parse_Rollup_hotfix(HKEY hKey, char *value) {
+    DWORD dataSize = MAXSTR;
+    LONG result;
+    
+    result = RegQueryValueEx(hKey, "InstallLocation", NULL, NULL, (LPBYTE)value, &dataSize);  
+    if (result != ERROR_SUCCESS ) {
+        mtdebug2(WM_SYS_LOGTAG, "Error reading 'InstallLocation' from Windows registry. (Error %u)",(unsigned int)result);
```

Comment:
I would change these debug messages to `debug1` if they are not flooding, or to `error/warning` if represent unexpected situations